notification/notification_test.go

```diff
@@ -50,7 +50,8 @@ func (m mockConfig) GetReceiveEventNotificationEndpoint() string {
 
 func TestWebhookSender_Send(t *testing.T) {
 	docID := utils.RandomSlice(32)
-	cid := utils.RandomSlice(32)
+	accountID := utils.RandomSlice(6)
```

Comment:
Why is the accountid so short?

Reply:
its a centID

Reply:
the account_id is the centID and as of today it is 6 bytes long.

Reply:
btw @mikiquantum could you use the `identity.CentIDLength` constant for this or use `identity.RandomCentID` function? better for readability and understanding.